CMakeLists.txt

```diff
@@ -723,6 +723,9 @@ set (HDF5_SRC_INCLUDE_DIRS
 )
 option (HDF5_ENABLE_SUBFILING_VFD "Build Parallel HDF5 Subfiling VFD" OFF)
 if (HDF5_ENABLE_SUBFILING_VFD)
+  if (WIN32)
```

Comment:
No this is not right, there is already a check:
-- Looking for stdatomic.h - not found
CMake Error at CMakeLists.txt:749 (message):
  Subfiling VFD requires atomic operations support.  C11 stdatomic.h NOT
  available.


This should disable subfiling.

Reply:
Intel MPI does support atomic, but not posix. Currently, subfiling does not work with windows. See https://jira.hdfgroup.org/browse/OESS-388

Reply:
I don't understand why this check is necessary - because the stdatomic.h check will fail on windows. Right?

Reply:
> I don't understand why this check is necessary - because the stdatomic.h check will fail on windows. Right?

https://devblogs.microsoft.com/cppblog/c11-atomics-in-visual-studio-2022-version-17-5-preview-2/